src/src/com/microsoft/aad/adal/Discovery.java

```diff
@@ -205,16 +209,7 @@ private boolean sendRequest(final URL queryUrl) throws MalformedURLException, JS
             }
             
             return (discoveryResponse != null && discoveryResponse.containsKey(TENANT_DISCOVERY_ENDPOINT));
-        } catch (IllegalArgumentException exc) {
```

Comment:
:disappointed: Thanks for cleaning this up. I don't understand why we did it this way, catch it and rethrow it, just for logging?!...